src/Features/VisualBasic/Portable/ChangeAccessibilityModifier/VisualBasicChangeAccessibilityModifierCodeFixProvider.vb

```diff
@@ -0,0 +1,34 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System.Collections.Immutable
+Imports System.Composition
+Imports System.Diagnostics.CodeAnalysis
+Imports Microsoft.CodeAnalysis
+Imports Microsoft.CodeAnalysis.ChangeAccessibilityModifier
+Imports Microsoft.CodeAnalysis.CodeFixes
+Imports Microsoft.CodeAnalysis.VisualBasic
+
+<ExportCodeFixProvider(LanguageNames.VisualBasic, Name:=PredefinedCodeFixProviderNames.ChangeAccessibilityModifier), [Shared]>
+Friend Class VisualBasicChangeAccessibilityModifierCodeFixProvider
+    Inherits AbstractChangeAccessibilityModifierCodeFixProvider
+
+    'BC31408: 'Private' and 'MustOverride' cannot be combined.
+    Private Const BC31408 As String = NameOf(BC31408)
+
+    'BC30266: 'accessibility identifier' cannot override 'accessibility identifier' because they have different access levels.
+    Private Const BC30266 As String = NameOf(BC30266)
+
+    <ImportingConstructor>
+    <SuppressMessage("RoslynDiagnosticsReliability", "RS0033:Importing constructor should be [Obsolete]", Justification:="Used in test code: https://github.com/dotnet/roslyn/issues/42814")>
+    Public Sub New()
+    End Sub
+
+    Public Overrides ReadOnly Property FixableDiagnosticIds As ImmutableArray(Of String) =
+        ImmutableArray.Create(BC31408, BC30266)
+
+    Protected Overrides Function GetText(accessibility As Accessibility) As String
```

Comment:
Why not use the `syntaxFacts` you got from `document.GetLanguageService<ISyntaxFactsService>();` in the abstract class and implement the method in the abstract class instead of having as abstract method?

Reply:
I viewed `ISyntaxFactsService` and `ISyntaxFacts`, and didn't find such member.